### PR TITLE
Consistent use of Target Language

### PR DESCRIPTION
**Pull request by prakharrathi25**
I have updated the display type of target language for a translation as requested by @pranav2305 

Fixes CL-50.

### CODE REVIEW

**Comment by GokulNC** [comment]
Hii @prakharrathi25 , this does not seem to have the custom migration to rename lang name to lang code.
Can you please recheck once?

File: backend/translation/serializers.py [hunk 1]
```diff
@@ -5,14 +5,10 @@
 
 class TranslationSerializer(serializers.ModelSerializer):
     translation_type = serializers.SerializerMethodField()
-    target_language = serializers.SerializerMethodField()
```
Comment (GokulNC) [inline backend/translation/serializers.py:8]: Why is this removed? Without the serializer having the language, how will the language be passed via the response?

Comment (prakharrathi25) [reply backend/translation/serializers.py:8]: Discussed on call.
